api/nakadi-event-bus-api.yaml

```diff
@@ -278,6 +278,9 @@ paths:
 
          * Events in the stream may be visible for a short period of time before being removed.
 
+         On nakadi-live, the oauth resource owner username has to be equal to 'nakadi.oauth2.adminClientId' property
```

Comment:
Good point. Would it be ok to replace this with `Depending on the nakadi configuration, the oauth resource owner username may have to be equal [...]`?